net-lib/net/dns.rkt

```diff
@@ -14,6 +16,28 @@
            (->* ((or/c ip-address? ip-address-string?) string?)
                 (#:ipv6? any/c)
                 ip-address-string?)]
+          [dns-get-srv
+           (->* ((or/c ip-address? ip-address-string?)
+                 string?
+                 string?)
+                (string?)
+                (listof (struct/c srv-rr
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  string?)))]
+          [dns-lookup
```

Comment:
It looks like `dns-lookup` is exported but undocumented? Should it be documented? (or unexported?)